Formula/libnghttp2.rb

```diff
@@ -1,10 +1,9 @@
 class Libnghttp2 < Formula
   desc "HTTP/2 C Library"
   homepage "https://nghttp2.org/"
-  url "https://github.com/nghttp2/nghttp2/releases/download/v1.48.0/nghttp2-1.48.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/nghttp2-1.48.0.tar.gz"
-  mirror "http://fresh-center.net/linux/www/legacy/nghttp2-1.48.0.tar.gz"
```

Comment:
Legacy mirror should stay for when this gets outdated. Note how http://fresh-center.net/linux/www/nghttp2-1.48.0.tar.gz is now 404 but the legacy mirror works.